src/loaders/config.py

```diff
@@ -6,21 +6,25 @@
     with open('config/config.yaml', 'r') as file:
         config = safe_load(file)
 except FileNotFoundError:
-    print('Config not found. Using package defaults instead.')
-    DATA_PATH = 'data'
-    NUM_OF_PAGES = 10
-    BATCH_SIZE = 1000
+    print('ERROR: config.yaml file not found!')
+    raise
```

Comment:
I think the better way to raise an error is `raise FileNotFoundError ('ERROR: config.yaml file not found!')`